contracts/EscrowAndStaking/EscrowThales.sol

```diff
@@ -159,34 +182,92 @@ contract EscrowThales is IEscrowThales, Initializable, ProxyOwned, ProxyReentran
         return true;
     }
 
+    /// @notice Set address of Staking Thales contract
+    /// @param StakingThalesContract address of Staking Thales contract
     function setStakingThalesContract(address StakingThalesContract) external onlyOwner {
         require(StakingThalesContract != address(0), "Invalid address set");
         iStakingThales = IStakingThales(StakingThalesContract);
         emit StakingThalesContractChanged(StakingThalesContract);
     }
 
+    /// @notice Enable the test mode
     function enableTestMode() external onlyOwner {
         testMode = true;
     }
 
+    /// @notice Set address of Airdrop contract
+    /// @param AirdropContract address of Airdrop contract
     function setAirdropContract(address AirdropContract) external onlyOwner {
         require(AirdropContract != address(0), "Invalid address set");
         airdropContract = AirdropContract;
         emit AirdropContractChanged(AirdropContract);
     }
 
+    /// @notice Set address of Thales staking rewards pool
+    /// @param _thalesStakingRewardsPool address of Thales staking rewards pool
     function setThalesStakingRewardsPool(address _thalesStakingRewardsPool) public onlyOwner {
         require(_thalesStakingRewardsPool != address(0), "Invalid address");
         ThalesStakingRewardsPool = IThalesStakingRewardsPool(_thalesStakingRewardsPool);
         emit ThalesStakingRewardsPoolChanged(_thalesStakingRewardsPool);
     }
 
+    /// @notice Fix the vesting entry for the account
+    /// @param account to fix the vesting entry for
     function fixEscrowEntry(address account) external onlyOwner {
         vestingEntries[account][currentVestingPeriod.mod(NUM_PERIODS)].vesting_period = currentVestingPeriod.add(
             NUM_PERIODS
         );
     }
 
+    /// @notice Merge account to transfer all escrow amounts to another account
+    /// @param srcAccount to merge
+    /// @param destAccount to merge into
+    function mergeAccount(address srcAccount, address destAccount) external {
+        require(msg.sender == address(iStakingThales), "Can only be called from staking contract");
+
```

Comment:
so if we want to deploy unactivate `mergeAccount()` function we would need a flag in the StakingThales
`require(iStakingThales.mergeAccountEnabled(), "Merge account disabled");`

Reply:
Actually since StakingThales is the only callee, this would not be necessary